project/Depend.scala

```diff
@@ -41,7 +41,12 @@ object Depend {
       "" -> Nil,
       "agent" -> List("api"),
       "api" -> List("core", "core/model", "util"),
-      "app" -> List("ide", "window"),
+      "app" -> List("app/codetab", "app/common", "app/infotab", "app/interfacetab", "app/tools"),
+      "app/codetab" -> List("app/common"),
+      "app/infotab" -> List("app/common"),
+      "app/interfacetab" -> List("app/common", "app/tools"),
```

Comment:
I realize the dependence on "app/tools" might not be desired. If not, let me know and I'll cut this dependency out (it's very easy to)